src/teleop/gui/src/components/ESWDebugGUI.vue

```diff
@@ -0,0 +1,224 @@
+  <div class="wrapper">
+    <div class="box header">
+      <img
+        src="/static/mrover.png"
+        alt="MRover"
+        title="MRover"
+        width="48"
+        height="48"
+      />
+      <h1>ESW Debug</h1>
+      <div class="spacer"></div>
+      <div class="spacer"></div>
+    </div>
+    <div class="box arm-controls light-bg">
+      <ArmControls />
+    </div>
+    <div class="box drive-vel-data light-bg">
+      <JointStateTable
+        :joint-state-data="motorJointState"
+        :vertical="true"
+        :header="'Drive Motors'"
+      ></JointStateTable>
+    </div>
+    <div class="box arm-jointstate light-bg">
+      <JointStateTable
+        :joint-state-data="armJointState"
+        :vertical="true"
+        :header="'Arm Motors'"
+      ></JointStateTable>
+    </div>
+    <div class="box drive-moteus light-bg">
+      <DriveMoteusStateTable
+        :moteus-state-data="driveMoteusState"
+      ></DriveMoteusStateTable>
+    </div>
+    <div class="box arm-moteus light-bg">
+      <MoteusStateTable
+        :moteus-state-data="armMoteusState"
+        :header="'Arm Moteus State'"
+      ></MoteusStateTable>
+    </div>
+    <div class="box pdb light-bg">
+      <PDBFuse></PDBFuse>
+    </div>
+    <div class="box cameras light-bg">
+      <Cameras :primary="primary"></Cameras>
+      <Checkbox
+        ref="Primary"
+        :name="'Primary Computer?'"
+        @toggle="primary =$event"
+      />
+    </div>
+    <div class="box velocity light-bg">
+      <Velocity></Velocity>
+    </div>
+    <div class="box odometry light-bg">
+      <Odom :odom="odom"></Odom>
+    </div>
+    <DriveControls></DriveControls>
+    <GimbalControls></GimbalControls>
+  </div>
+</template>
+
+<script>
+import ROSLIB from "roslib";
+import DriveControls from "./DriveControls.vue";
+import ArmControls from "./ArmControls.vue";
+import GimbalControls from "./MastGimbalControls.vue";
+import JointStateTable from "./JointStateTable.vue";
+import DriveMoteusStateTable from "./DriveMoteusStateTable.vue";
+import MoteusStateTable from "./MoteusStateTable.vue";
+import PDBFuse from "./PDBFuse.vue";
+import Cameras from "./Cameras.vue";
+import Checkbox from "./Checkbox.vue";
+import Odom from "./OdometryReading.vue";
+import Velocity from "./VelocityCommand.vue";
+
+export default {
+  components: {
+    DriveControls,
+    ArmControls,
+    GimbalControls,
+    JointStateTable,
+    DriveMoteusStateTable,
+    MoteusStateTable,
+    PDBFuse,
+    Cameras,
+    Checkbox,
+    Odom,
+    Velocity
+  },
+  data() {
+    return {
+      primary: false,
+
+      // Default object isn't empty, so has to be initialized to ""
+      driveMoteusState: {
+        name: ["", "", "", "", "", ""],
+        error: ["", "", "", "", "", ""],
+        state: ["", "", "", "", "", ""]
+      },
+
+      armMoteusState: {
+        name: ["", "", "", "", "", ""],
+        error: ["", "", "", "", "", ""],
+        state: ["", "", "", "", "", ""]
+      },
```

Comment:
It may be nice to put this in the MoteusStateTable component so that we don't have to have this duplication with this and with the task components.